KubeArmor/core/kubeUpdate.go

```diff
@@ -959,48 +959,6 @@ func (dm *KubeArmorDaemon) WatchK8sPods() {
 	}
 }
 
-func matchClusterSecurityPolicyRule(policy tp.SecurityPolicy) bool {
-	hasInOperator := false
```

Comment:
let's add a check here :
```
if len(policy.Spec.Selector.Identities) >0 {
		return false
	}
```
this would indicate that the policy is a `KubeArmorSecurityPolicy`, and not the cluster policy.

Reply:
cc : @daemon1024

Reply:
I have updated the commit. Thank you @Prateeknandle!